Update active-directory-mobileanddesktopapp-ios-introduction.md

JohnMAustin78 opened the pull request:
Developers need to know that they cannot use the MSAL library for iOS with the MS Graph client library before they read the article and try to retrofit their client library based apps with MSAL.

azurecla commented:
Hi __@AzureDocsBot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jodoglevy commented:
Please ignore - this is a test for HackADoc

Commit:
Update active-directory-mobileanddesktopapp-ios-introduction.md

Developers need to know that they cannot use the MSAL library for iOS with the MS Graph client library before they read the article and try to retrofit their client library based apps with MSAL.

andretms reviewed (commented):
I would recommend not committing this change in this guide: the guided setup uses REST APIs to query graph already and that is the recommendation. If customers want to use Graph API SDKs, they can write their adapter to work using REST APIs.
